third_party/nvfuser/csrc/python_frontend/fusion_definition.h

```diff
@@ -192,7 +204,7 @@ class TORCH_CUDA_CU_API FusionDefinition {
   struct Operators {
     Operators(FusionDefinition* fd) : fusion_definition(fd) {}
     bool validUse() const {
-      return !fusion_definition->id().has_value();
+      return !fusion_definition->completed();
```

Comment:
@kevinstephano I changed this so that `FusionDefinition` can tell us if it's completed. I needed that since I bypass `Operators` when defining methods for `Tensor` and `Scalar`, and grab the `FusionDefinition` directly from them.